routes/dish.js

```diff
@@ -26,25 +28,39 @@ router.get("/", (req, res) => {
     .catch((err) => res.status(404).json(err));
 });
 
-router.post("/", upload.single("image"), (req, res) => {
-  dishAPI
-    .create(req)
-    .then((dish) => res.json(dish))
-    .catch((err) => res.json(err));
-});
+router.post(
+  "/",
+  upload.single("image"),
+  passport.authenticate("jwt", { session: false }),
```

Comment:
When do you need to call this? I see you use it 4 times in this PR

Reply:
Just because I havent added it everywhere. That's the backend auth to check if someone is a manager

Reply:
Then I think it would be smart to use this pattern for it, instead of adding it manually to many routes:
https://stackoverflow.com/a/25603056/1380370
Tell me if you agree

Reply:
Agree! much more elegant